src/views/preview/presentation.jsx

```diff
@@ -1,332 +1,416 @@
-                            <div className="comments-container">
-                                <div className="project-title">
-                                    Comments go here
-                                </div>
-                            </div>
-                            <FlexRow className="column">
-                                <FlexRow className="remix-list">
+                            <FlexRow className="preview-row">
+                                <div className="comments-container">
                                     <div className="project-title">
-                                        Remixes
+                                        Comments go here
                                     </div>
-                                    {remixes && remixes.length === 0 ? (
-                                        <span>No remixes</span>
-                                    ) : (
-                                        <ThumbnailColumn
-                                            cards
-                                            showAvatar
-                                            itemType="preview"
-                                            items={remixes.slice(0, 5)}
-                                            showFavorites={false}
-                                            showLoves={false}
-                                            showViews={false}
-                                        />
+                                </div>
+                                <FlexRow className="column">
+                                    {/* hide remixes if there aren't any */}
+                                    {remixes && remixes.length !== 0 && (
+                                        <FlexRow className="remix-list">
+                                            <div className="project-title">
+                                                Remixes
+                                            </div>
+                                            {remixes && remixes.length === 0 ? (
+                                                // TODO: style remix invitation
+                                                <span>Invite user to remix</span>
+                                            ) : (
+                                                <ThumbnailColumn
+                                                    cards
+                                                    showAvatar
+                                                    itemType="preview"
+                                                    items={remixes.slice(0, 5)}
+                                                    showFavorites={false}
+                                                    showLoves={false}
+                                                    showViews={false}
+                                                />
+                                            )}
+                                        </FlexRow>
```

Comment:
If this were a component it could decide to show/hide based on the content of the `remixes` prop.